### PR TITLE
S16FRANCIS-32: Basic end to end framework (Angular UI, Spring MVC Pro…

### PR DESCRIPTION
**Pull request by akshaykamath**
This is a basic integrated framework that uses angular ui, spring mvc rest, junit and bootstrap.

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
```
Comment (burnumd) [inline .classpath:2]: Don't commit .classpath files

File: .project [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```
Comment (burnumd) [inline .project:2]: Don't commit IDE files

**Comment by burnumd** [comment]
Remove all the `.settings/**` files and add them to .gitignore (in addition to the `.classpath` and `.project` files).

File: src/main/resources/config.properties [hunk 3]
```diff
@@ -0,0 +1,2 @@
+test.username=user
```
Comment (burnumd) [inline src/main/resources/config.properties:1]: This file shouldn't be committed if it has passwords in it (even if this is a dummy password, someone might change it on their local machine and accidentally commit it).

File: src/main/webapp/WEB-INF/integrator-rest.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+http://www.springframework.org/schema/mvc
+http://www.springframework.org/schema/mvc/spring-mvc.xsd">
+
+	<context:component-scan base-package="org.p632.turnkey.*" />
+	<mvc:annotation-driven />
+
+	<bean id="propertyConfigurer"
+		class="org.springframework.context.support.PropertySourcesPlaceholderConfigurer">
+		<property name="location">
+			<value>classpath:config.properties</value>
```
Comment (burnumd) [inline src/main/webapp/WEB-INF/integrator-rest.xml:17]: If this file is going to contain passwords, it shouldn't be on the classpath.

File: src/main/webapp/css/bootstrap-theme.min.css [hunk 5]
```diff
@@ -0,0 +1,6 @@
+/*!
```
Comment (burnumd) [inline src/main/webapp/css/bootstrap-theme.min.css:1]: Minified front-end files should be treated like Java .class files (i.e., they shouldn't be under VCS). You might consider looking at something like Webpack or Gulp to generate these files on the fly.

**Comment by burnumd** [comment]
The Code Climate for this branch is pretty abysmal: https://codeclimate.com/github/p632-sp-2016/continuous-delivery/compare/feature/S16FRANCIS-32

**Comment by akshaykamath** [comment]
S16FRANCIS33 has been raised to address the issues raised in this merge request. Code climate gpa is low due to external package dependencies. This can be addressed by using npm, bower in S16FRANCIS33. Merging the pull request.

**Pull request merged**